### ISSUE TITLE
Add Issue descriptions to issue tab

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/network-pulse#204]
I suggested that when a user navigates to the Issues tab, they should see a statement about Mozilla's issues (rather than loading the first issue). I'm not sure if I agree with myself. The experience is okay on mobile, but feels empty and maybe broken on desktop. The issue buttons also feel ungrounded without item cards below. 

It could probably be fixed with different UX or UI. Let's rethink this.

**Comment by kristinashu** [comment]
We could preselect the first issue so there are some posts within view (moved the hint text above the issue tags):
![02](https://cloud.githubusercontent.com/assets/1068783/20403161/9305975e-acb4-11e6-9407-aec945fbe873.png)

Or, we could show all the issues (like latest) and then only once a user selects a tag do we show the filtered down view.
![01](https://cloud.githubusercontent.com/assets/1068783/20403160/92e500ac-acb4-11e6-95c8-c3606ae3d6b8.png)

**Comment by xmatthewx** [comment]
- I like how you now have the text above the issue buttons. 
 - I like how the first option highlights one, and draws some attention to those buttons. 
 - Maybe we can randomly select one on load

**Comment by kristinashu** [comment]
Cool, let's go with option 1 were the issue is randomly selected. 

![9305975e-acb4-11e6-9407-aec945fbe873](https://cloud.githubusercontent.com/assets/1068783/20448515/3f412076-ad99-11e6-8666-dc24c9fb9cc2.png)

**Comment by mmmavis** [comment]
I mentioned this in my [reactification plan](https://docs.google.com/document/d/1a1ldcWM5ow60cyz9IosuAJGuTmhyDX7zpCxZSmXJgbg/edit#)

Are we good with the following paths?

- Issues tab: /issues
  - Online Privacy & Security
    - /issues/online-privacy-and-security
  - Open Innovation
    - /issues/open-innovation
  - Decentralization
    - /issues/decentralization
  - Web Literacy
    - /issues/web-literacy
  - Digital Inclusion
    - /issues/digital-inclusion

If we go with option 1 with one issue randomly preselected this means /issues will be *randomly* redirected.... which can be really odd. Thoughts?

/cc @kristinashu @xmatthewx @alanmoo @Pomax

**Comment by xmatthewx** [comment]
You're right, that would be odd. 

We could continue to use our Issues landing page as currently implemented, adding descriptions for each issue like these: https://www.mozilla.org/en-US/foundation/issues/

Would be nice to have two columns on desktop, instead of full width paragraphs. If the Insights team develops a graphic for each image, we could add those. For now, maybe something like so...


<img width="1171" alt="screenshot 2016-12-15 13 22 23" src="https://cloud.githubusercontent.com/assets/53411/21236566/8f904bde-c2c9-11e6-8b1d-383187dfb1ea.png">

**Comment by alanmoo** [comment]
If we go with this approach, I already have the description for each issue in the database, so you don't have to manually paste them in to a route @mmmavis.

**Comment by kristinashu** [comment]
Updated the look of this a little. I'll open a new ticket for implementation.

![desktop - issues landing 2x](https://cloud.githubusercontent.com/assets/1068783/25670476/a475b888-2fe1-11e7-93ed-f0982adfc036.png)
